foundation-extensions/places/README.md

```diff
@@ -1,13 +1,179 @@
 
 The SDK extension listens for events that contain GPS coordinates and geofence region events, and dispatches new events that are processed by the Rules Engine. The SDK extension also retrieves and delivers a list of the nearest POI for the app data that retrieves from the APIs. The regions returned by the APIs are stored in cache and persistence, which allows limited offline processing.
 
-To get started with implementing Adobe Experience Platform Places Service on your mobile app, review documentation on [Places extension](https://experienceleague.adobe.com/docs/places/using/places-ext-aep-sdks/places-extension/places-extension.html).
+`Places` is the mobile SDK supporting the Location Service.
 
-## Reference
+## Add the AEP Places extension to your app
 
-* [AEPPlaces API reference](places-usage-reference.md)
-* [Migration to AEPPlaces](../../resources/migrate-to-swift.md) - Migrate your current Objective-C implementation to Swift-based SDK libraries
+### Download and import the Places extension
 
+{% tabs %}
+{% tab title="Android" %}
+### Java
+
+1. Add the Mobile Core and Places extensions to your project using the app's Gradle file.
+
+```java
+implementation 'com.adobe.marketing.mobile:core:1.+'
+implementation 'com.adobe.marketing.mobile:places:1.+'
+```
+
+2. Import the Mobile Core and Places extensions in your Application class.
+
+```java
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Places;
+```
+
+{% endtab %}
+
+{% tab title="iOS (AEP 3.x)" %}
+
+1. Add the Mobile Core and Places extensions to your project using CocoaPods. Add the following pods in your `Podfile`:
+
+```swift
+use_frameworks!
+target 'YourTargetApp' do
+   pod 'AEPCore'
+   pod 'AEPPlaces'       
+end
+```
+
+2. Import the Mobile Core and Places modules:
+
+### Swift
+
+```swift
+// AppDelegate.swift
+import AEPCore
+import AEPPlaces
+```
+
+### Objective-C
+
+```objectivec
+// AppDelegate.h
+@import AEPCore;
+@import AEPPlaces;
+```
+{% endtab %}
+
+{% tab title="iOS (ACP 2.x)" %}
+
+1. Add the Mobile Core and Places extensions to your project using CocoaPods. Add the following pods in your `Podfile`:
+
+```swift
+use_frameworks!
+target 'YourTargetApp' do
+   pod 'ACPCore', '~> 2.0'
+   pod 'ACPPlaces', '~> 1.0'
+end
+```
+
+2. Import the Mobile Core and Places modules:
+
+### Swift
+
+```swift
+// AppDelegate.swift
+import ACPCore
+import ACPPlaces
+```
+
+### Objective-C
+
+```objectivec
+// AppDelegate.h
+#import "ACPCore.h"
+#import "ACPPlaces.h"
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+### Register the Places extension with Mobile Core
+
+{% tabs %}
+{% tab title="Android" %}
+### Java
+
+```java
+public class MobileApp extends Application {
+
+    @Override
+    public void onCreate() {
+      super.onCreate();
+      MobileCore.setApplication(this);
+      MobileCore.configureWithAppID("yourLaunchEnvironmentID");
```

Comment:
can we remove the launch reference here. and do something like
MobileCore.configureWithAppID("yourAppID");